esx_service/vmdk_ops.py

```diff
@@ -658,8 +658,8 @@ def reset_vol_meta(vmdk_path):
     vol_meta[kv.ATTACHED_VM_UUID] = None
     vol_meta[kv.ATTACHED_VM_NAME] = None
     if not kv.setAll(vmdk_path, vol_meta):
-       logging.warning("reset_vol_meta: Failed to save Disk metadata for %s", vmdk_path)
-       msg = "Failed to reset stale metadata for {0}.".format(vmdk_path)
+       logging.warning("reset_vol_meta: Failed to save volume metadata for %s", vmdk_path)
+       msg = "Failed to save volume metadata for {0}.".format(vmdk_path)
```

Comment:
Thanks for the review. Made the change.

On Thu, Aug 18, 2016 at 12:16 PM, Mark Sterin <notifications@github.com>
wrote:

> In esx_service/vmdk_ops.py
> <https://github.com/vmware/docker-volume-vsphere/pull/573#discussion_r75255874>
> :
>
> > @@ -658,8 +658,8 @@ def reset_vol_meta(vmdk_path):
> >      vol_meta[kv.ATTACHED_VM_UUID] = None
> >      vol_meta[kv.ATTACHED_VM_NAME] = None
> >      if not kv.setAll(vmdk_path, vol_meta):
> > -       logging.warning("reset_vol_meta: Failed to save Disk metadata for %s", vmdk_path)
> > -       msg = "Failed to reset stale metadata for {0}.".format(vmdk_path)
> > +       logging.warning("reset_vol_meta: Failed to save volume metadata for %s", vmdk_path)
> > +       msg = "Failed to save volume metadata for {0}.".format(vmdk_path)
>
> I still do not get why do you use copy-n-pasted strings. I was expecting
> simply
>
> msg = "Failed to save volume metadata for {0}.".format(vmdk_path)
> logging.warning("reset_vol_meta: " + msg)return err(msg)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/573/files/1e51c0cb1b40cb20d43f8a59a3ce5aab76f3f8c2..63ba4dbadb7bb321c504ec22356459fda5b19218#r75255874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APHseMafst2Hlyaf2epLtglz27J9yqNzks5qg_-4gaJpZM4JiB4U>
> .
>